11_install_agent/20_install_agent_rhel.txt

```diff
@@ -0,0 +1,39 @@
+=== Install Rudder Agent on RedHat-like systems
+
+Download the package applicable to your version of RedHat/CentOS and to its
+architecture on 
+
+----
+
+http://www.rudder-project.org/rpm-3.0/RHEL_<version>/
+
+----
+
+Or you can define a yum repository, replacing <version> with your actual EL version:
+
+----
+
+$ echo "[Rudder_3.0]
+name=Rudder 3.0 EL repository
+baseurl=http://www.rudder-project.org/rpm-3.0/RHEL_<version>/
+gpgcheck=1
+gpgkey=http://www.rudder-project.org/rpm-3.0/RHEL_<version>/repodata/repomd.xml.key" > /etc/yum.repos.d/rudder.repo
+
+----
+
+Install the package:
+
+----
+
+rpm -Uhv rudder-agent-3.0.0-1.EL.5.x86_64.rpm
+
+----
+
+Or if a yum repository has been set:
+
+----
+
+yum install rudder-agent
```

Comment:
These commands should be grouped with the various options above. It is weird to have method 1 step 1, method 2 step1, method 1 step 2, method 2 step 2.